wikichange/assets/content.css

```diff
@@ -3,15 +3,16 @@ mark {
     color: inherit;
 }
 
-button {
+button.extensionButton {
```

Comment:
made this into a button with class extensionButton so that the CSS doesn't apply to other buttons in wiki page